codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -181,12 +181,12 @@ update these file paths accordingly.
       processManager: processManager,
     );
 
-    // notarize
-    await notarize(codesignedFile);
-
     // `dryrun` flag defaults to true to prevent uploading artifacts back to google cloud.
     // This would help prevent https://github.com/flutter/flutter/issues/104387
     if (!dryrun) {
+      // notarize
```

Comment:
This comment isn't providing any value to the following line

Reply:
True, deleted.